src/rlx_prv_relup.erl

```diff
@@ -65,6 +65,15 @@ format_error({relup_script_generation_error,
                {missing_sasl, _}}}) ->
     "Unfortunately, due to requirements in systools, you need to have the sasl application "
         "in both the current release and the release to upgrade from.";
+format_error({relup_script_generation_warn, systools_relup,
+               [{erts_vsn_changed, _},
+                {erts_vsn_changed, _}]}) ->
+    "It has been detected that the ERTS version changed while generating the relup between versions, "
+    "please be aware that an instruction that will automatically restart the VM will be inserted in "
+    "this case";
+format_error({relup_script_generation_warn, Module, Warnings}) ->
+    ["Warnings generating relup \n",
+     rlx_util:indent(2), Module:format_warning("~p", [Warnings])];
```

Comment:
`Module:format_warning` takes only one argument - so it should be `Module:format_warning(Warnings)`.